test/Summary.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('The Summary', function() {
+    let page;
+    before(async function() {
+        page = await browser.newPage();
+        await page.goto('http://localhost:8080');
+    });
+
+    after(async function() {
+        await page.close();
+    });
+
+    describe('Initially', function() {
+
+        it("it shouldn't exist until the Sidebar component is shown", async function() {
```

Comment:
Just "should not exist" would suffice as a description give the block the test is in. See my example above.